src/clipboard.py

```diff
@@ -1,13 +1,21 @@
 import copy
 
+from gi.repository import Adw, GObject, Gtk
+
 from graphs import graphs, ui
 
 
-class BaseClipboard:
-    def __init__(self, application):
-        self.application = application
-        self.clipboard = []
-        self.clipboard_pos = -1
+class BaseClipboard(GObject.Object):
```

Comment:
Any reason to let the Clipboard specifically inherit from GObject?

Reply:
Since we currently only ever pickle the `clipboard` and `clipboard_pos` attribute which are a dict and an int respecively that isn't an issue.

Reply:
I was just thinking that `clipboard = GObject.Property(type=object)` kinda changes the type. 

I'm perfectly fine merging this, as it's a good change overall. I was just wondering if there's any benefit to having the Clipboard class inheriting from GObject as well, as opposed to keeping it "Vanilla Python".

Reply:
Technically it kind of does. However during get/set pygobject automatically handles the conversion for us.

As for benefit, I was planning an upcomming pr making use of GObjects Signal functionality and maybe also a huge refactor involving items trying to reduce unnecessary refreshs (also making use of some of this functionality).
The other reason is consistency. GObject has this really nic feature, where you define all properties with default values and can just pass them along during init. This is already used by pretty much all classes that depend on it, but isn't by the custom ones we created.